src/reducers/index.js

```diff
@@ -157,3 +177,16 @@ export function getTasksErrorMessage(state) {
 export function isFetchingTasks(state) {
   return taskSelectors.isFetchingTasks(state.tasks);
 }
+
+export function getSecrets(state) {
```

Comment:
should also accept namespace as an optional param like the other collection selectors in this file

Reply:
on it!